extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -185,6 +192,28 @@ export class LocalSubscriptionPlatform {
           return entitlement;
         });
   }
+
+  /** @override */
+  isPingbackEnabled() {
+    return this.isPingbackEnabled_;
+  }
+
+  /** @override */
+  pingback() {
+    const promise = this.urlBuilder_.buildUrl(this.pingbackUrl_,
+        /* useAuthData */ true);
+    return promise.then(url => {
+      dev().fine(this.getServiceId(), 'Pingback URL: ', url);
+      return this.xhr_.sendSignal(url, {
+        method: 'POST',
+        credentials: 'include',
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
B/c of the "raw" field, let's make this `text/plain`. That also gets rid of CORS preflight which is very important.